ui/app/src/Utils/Views.elm

```diff
@@ -6,12 +6,12 @@ import Html.Events exposing (onCheck, onInput, onClick)
 import Http exposing (Error(..))
 
 
-labelButton : Maybe msg -> ( String, String ) -> Html msg
-labelButton maybeMsg ( key, value ) =
+labelButton : Maybe msg -> String -> Html msg
+labelButton maybeMsg labelText =
```

Comment:
👍 for `labelText`. I think I changed it to `(key, value)` in my previous PR, sorry about that.